pkg/tnf/handlers/readremotefile/readremotefile.go

```diff
@@ -43,7 +42,7 @@ func NewReadRemoteFile(timeout time.Duration, nodeName, filePath string) *ReadRe
 		timeout: timeout,
 		result:  tnf.ERROR,
 		args: []string{
-			"echo", "\"cat /host" + filePath + "\"", "|", common.GetOcDebugCommand(), "-q", "node/" + nodeName,
+			"echo", "\"cat /host" + filePath + "\"", "|", "oc", "debug", "-q", "node/" + nodeName,
```

Comment:
Actually, I think this readremotefile handler could be completely removed, I can't find any place using it. However, let's do that post-3.2

Reply:
Yes, we have quite a few handlers that should have been removed/deprecated.